ledger/internal/compactcert_test.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/algorand/go-algorand/test/partitiontest"
 )
 
+// TODO Stateproof: need to update this test to use some proper message to verify with the compact cert
```

Comment:
we switch to a new approach when the message content does not have any meaning for the verifier of the transaction. 
the only thing that matters is that the cert attest to this specific message.
so I'm not sure that we need to check a meaningful message

Reply:
So basically we don't care what the message is, only that the compact cert correctly verifies this message. I'll add a verification of the message since there isn't any in the test though.